docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,122 @@
+3. After a new batch of data is loaded into the warehouse, a previously created Analysis Services tabular model is refreshed. This provides a semantic model that simplifies the analysis of business data and relationships by business analysts.
+4. Business analysts use Power BI to analyze data in the warehouse via the Analysis Services semantic model.
+
+### Components
+
+* [SQL Server](/sql/sql-server) databases containing source data may be located on-premises or in cloud-hosted virtual machines.
+* [Azure SQL Database](/azure/sql-database) is a general-purpose relational database managed service in Microsoft Azure that can provide data to a consolidated data warehouse.
+* [Cosmos DB](/azure/cosmos-db/) is a globally distributed, multi-model database service that can provide data to a consolidated data warehouse.
+* [Blob storage](/azure/storage/blobs) provides a staging area for the source data prior to loading it into SQL Data Warehouse.
+* [Data Factory](/azure/data-factory) orchestrates the transformation of staged data into a common structure in SQL Data Warehouse.
+* [SQL Data Warehouse](/azure/sql-data-warehouse) is a distributed system for performing analytics on large data. Its use of massive parallel processing (MPP) makes it suitable for running high-performance analytics. SQL Data Warehouse can use [PolyBase](/sql/relational-databases/polybase/polybase-guide) to rapidly load data from Blob storage.
+* [Analysis Services](/azure/analysis-services) provides a semantic model for your data and can increase system performance when analyzing your data. 
+* [Power BI](/power-bi) Power BI is a suite of business analytics tools to analyze data and share insights. Power BI can query a semantic model stored in Analysis Services, or it can query SQL Data Warehouse directly.
+* [Azure Active Directory (Azure AD)](/azure/active-directory/) authenticates users who connect to the Analysis Services server through Power BI. Data Factory can also use Azure AD to authenticate to SQL Data Warehouse via a service principal or Managed Service Identity (MSI).
+
+### Alternatives
+
+* This example demonstrates using an on-premises SQL Server database and an external dataset as the data sources to integrate via the pipeline. However, this architecture is suitable for loading a wide range of both relational and non-relational data sources.
+* Data Factory is designed for automating your data pipeline workflows. For jobs requiring high performance, considering [using Data Factory with Polybase](/azure/data-factory/connector-azure-sql-data-warehouse#use-polybase-to-load-data-into-azure-sql-data-warehouse). For one-time or on-demand jobs, you could also use available tools like SQL Server bulk copy (bcp) and AzCopy. For a general comparison, see [Choosing a data pipeline orchestration technology in Azure](/azure/architecture/data-guide/technology-choices/pipeline-orchestration-data-movement).
```

Comment:
What does "high performance" mean in this context?

Reply:
My understanding is that ADF uses PolyBase. We shouldn't imply that ADF is not high performance. - it's a pipeline orchestrator, it doesn't do the actual work.

AzCopy and PolyBase are for two different things. Use AzCopy to move data from on-prem into Storage, use PolyBase to load from Storage into SQL DW

We don't recommend bcp for large data transfers.